appveyor.yml

```diff
@@ -28,11 +28,11 @@ before_build:
   - cd c:\projects\g3log\
   - mkdir build
   - cd build 
-  - cmake -DCMAKE_BUILD=Relese -G "Visual Studio 14 2015 Win64" -DADD_G3LOG_UNIT_TEST=ON ..
-  - msbuild g3log.sln
+  - cmake -G "Visual Studio 14 2015 Win64" -DADD_G3LOG_UNIT_TEST=ON -DUSE_DYNAMIC_LOGGING_LEVELS=ON -DCHANGE_G3LOG_DEBUG_TO_DBUG=ON ..
+  - msbuild g3log.sln /p:Configuration=Release
   
 # scripts to run after build
 after_build:
-  - cd c:\projects\g3log\build\Release
-  - g3log-FATAL-contract.exe
+  - cmd /c Release\g3log-FATAL-contract.exe || exit /B 0
+  - msbuild RUN_TESTS.vcxproj /p:Configuration=Release
```

Comment:
Cool thanks.

When this is working later my hope is that appveyor will run all the unit tests. 

Thanks for the DEBUG to DBUG change too.  Great to capture this on this CI